source-code/sdk-js/src/adapter-sveltekit/vite-plugin/transforms/refactoring.md

```diff
@@ -209,27 +427,181 @@ JavaScript files can run on the server and on the client. So we need to detect t
       // result => 'import { get } from "svelte/store";const fn = () => { const { i } = getRuntimeFromContext();get(i)("test") }'
       ```
 
+Possible implementation:
+
+```js
+function transformJs(ast, config) {
+  const aliases = replaceSdkImports(ast)
+	aliases.forEach(({exportN, aliasN}) =>
+    // Wrap all occurences of `i`, `language`, etc with `get()`
```

Comment:
only `i` and `language`

Reply:
This is resolved with the changed strategy